ui/src/app/shared/service/utils.ts

```diff
@@ -487,6 +487,9 @@ export class Utils {
    * @param value the value to convert
    */
   public static roundSlightlyNegativeValues(value: number) {
+    if (value === null) {

```

Comment:
even this. '==' instead of '==='. Based on the last discussion?

Reply:
ja 🤔 did i not push?